docs/manual/tutorial.md

```diff
@@ -221,6 +224,77 @@ and `System.exit_filters(self, filters, entity)`, so you can override
 it. `filters` is the list of names of `Filters` to be entered/exited.
 
 
+### Proxying Component Types
+
+When developing a system, it is entirely possible that you don't know
+what field of which component it will be working on. For example, when
+developing a camera mechanic, you may neither know nor care what type
+it should be aimed at, just that there is a `NodePath` that the camera
+should be looking at, from, following, whatever the system does.
+
+To do that, the component type and field where the `NodePath` can be
+found can be proxied. In an `entity_filter`, `Proxy` can be used as a
+stand-in. A name is given to it to look up the actual type to use later.
+
+```python
+from wecs.core import System
+from wecs.core import Proxy
+from wecs.panda3d.camera import Camera
+
+
+class MyAbstractCamera(System):
+    entity_filters = {
+        'camera': and_filter(Camera, Proxy('camera_focus')),
+    }
+```
+
+When writing a game that uses that camera system, the component type
+will of course be known. Let's say we use the prototype mechanics, and
+want to center the camera on the visible geometry:
+
+```python
+from wecs.panda3d.prototype import Geometry
+
+
+class MyNonAbstractCamera(MyAbstractCamera):
+    proxies = {
+        'camera_focus': ProxyType(Geometry, 'node'),
+    }
+```
+
+This system's `camera` filter will now match all entities with the
+`Camera` and `Geometry` components. This syntax is helpful if you want
+to provide defaults, but more compact way to specify proxies is to pass
+them as keyword arguments:
+
```

Comment:
...as a keyword arguments -> "as keywords arguments when creating the system"
And I think the code example should include a the entire 
```
system_types = [
```
part to show content for this.